doc/rest.md

```diff
@@ -495,6 +500,11 @@ Content-Type: `application/json`
 JSON object with following structure:
 - **name**: String. Mapname used for identification within Layman user workspace. Equal to `name` attribute of JSON root object
 - **uuid**: String. UUID of the map.
+- **layman_metadata**
+  - **publication_status**: String. Can be one of these values:
```

Comment:
Mention in POST Maps:

```
Response to this request may be returned sooner than the processing chain is finished to enable asynchronous processing. Status of processing chain can be seen using [GET Workspace Map](#get-workspace-map) and **status** properties of layer sources (file, thumbnail, metadata).
```

Mention also publication_status for watching global state